hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/wrapper/MessageContentWrapperProvider.java

```diff
@@ -0,0 +1,27 @@
+package pl.allegro.tech.hermes.common.message.wrapper;
+
+import pl.allegro.tech.hermes.api.Topic;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MessageContentWrapperProvider {
+
+    private final Map<Topic.ContentType, MessageContentWrapper> wrappers = new HashMap<>(2);
+    private final JsonMessageContentWrapper jsonMessageContentWrapper;
```

Comment:
Mayby we could pick somewhere default Topic.ContentType and choose MessageContentWrapper according this ContentType?